lib/shopt.sh

```diff
@@ -27,10 +27,17 @@ shopt -s nocaseglob;
 ## SMARTER TAB-COMPLETION (Readline bindings) ##
 
 # Perform file completion in a case insensitive fashion
-bind "set completion-ignore-case on"
-
-# Treat hyphens and underscores as equivalent
-bind "set completion-map-case on"
+if [ ${CASE_SENSITIVE:-false} != "true" ]
+then
+  bind "set completion-ignore-case on"
+
+  # Treat hyphens and underscores as equivalent
+  # CASE_SENSITIVE must be off
+  if [ ${HYPHEN_INSENSITIVE:-false} = "true" ]
```

Comment:
Although ohmyzsh has the setting with the name `HYPHEN_INSENSITIVE`, I feel we could use `HYPHEN_SENSITIVE` to `HYPHEN_"IN"SENSITIVE` because it is consistent with `CASE_SENSITIVE` and also negative boolean variables should be generally avoided. Or another way is to use the names like `IGNORE_CASE` and `IGNORE_HYPHEN` (and revert the meaning of `CASE_SENSITIVE`).

Reply:
I have changed the variable name to `HYPHEN_SENSITIVE`.

I decided to keep `SENSITIVE` instead of switching to `IGNORE` for now.